akka-cluster/src/main/scala/akka/cluster/Member.scala

```diff
@@ -345,6 +345,23 @@ final case class UniqueAddress(address: Address, longUid: Long) extends Ordered[
    */
   @deprecated("Use Long UID constructor instead", since = "2.4.11")
   @nowarn("msg=deprecated")
-  def copy(address: Address = address, uid: Int = uid) = new UniqueAddress(address, uid.toLong)
+  def copy(address: Address, uid: Int) = new UniqueAddress(address, uid.toLong)
+
+  /**
+   * For binary compatibility
+   * Stops `copy(Address)` copy from being generated, use `apply` instead.
+   */
+  @deprecated("Use Long UID constructor instead", since = "2.4.11")
+  @nowarn("msg=deprecated")
+  def copy(address: Address) = new UniqueAddress(address, uid.toLong)
+
+  /**
+   * For binary compatibility
+   * Stops `copy(Long)` copy from being generated, use `apply` instead.
+   */
+  @deprecated("Use Long UID constructor instead", since = "2.4.11")
+  @nowarn("msg=deprecated")
+  def copy(uid: Int) = new UniqueAddress(address, uid.toLong)
```

Comment:
Here is an example of the copy method I mentioned in Slack. In the case, it's easy to overload the methods.

We have similar problems with ClusterRouterConfig and ClusterRouterGroupSettings

Reply:
Looks like one option is to convert the case class into a regular class like in https://github.com/akka/akka/pull/29956/commits/cca8f71602438f886cc50b994e35131897be3777 . Painful though.